From ab3545544602085a777924f244f5aab914017c43 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:43:22 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(f69310a)..lineage-16.0-android-9.0.0_r31(9b0f3dd) (DO NOT
 SUBMIT)

Android Build Merger (Role) (2):
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   f678839035
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   f678839035 skipped: 9d959be951

Bill Yi (7):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   f678839035 skipped: 9d959be951 am: a9f73101a2  -s ours
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   f678839035 skipped: 9d959be951 am: a9f73101a2  -s ours am: 5b946d3f8e
    -s ours
 - [automerger skipped] Import translations. DO NOT MERGE skipped:
   f678839035 skipped: 9d959be951 am: a9f73101a2  -s ours am: 5b946d3f8e
    -s ours am: 631a40ebcf  -s ours

Chad Brubaker (1):
 - Ensure all android.process.media apps have usesCleartextTraffic=true

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (4):
 - Snap for 5044688 from 4157477bcec29fc56f28b5dcdcc1d4defcc61cea to
   pi-qpr2-release
 - Snap for 5073521 from 047e39bb78384db8981c83b92063db34d758c912 to
   pi-qpr2-release
 - Snap for 5115288 from 46a694cd2149b9feb5b21b08e97d0b30cc014227 to
   pi-qpr2-release
 - Snap for 5130385 from fa425c6626628d23e6e8cd14c32ab90a21868201 to
   pi-qpr2-release

Change-Id: Ia613ee94df49996fe8c861fb23745d9f522833f8

Change-Id: Ia3704fc54cab766473b410850cca5c18f59e0e07
---
 res/values-es/strings.xml     | 2 +-
 res/values-iw/strings.xml     | 2 +-
 res/values-pt-rPT/strings.xml | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/values-es/strings.xml b/res/values-es/strings.xml
index 1588eb8..6fa3d45 100644
--- a/res/values-es/strings.xml
+++ b/res/values-es/strings.xml
@@ -22,7 +22,7 @@
     <string name="artist_label" msgid="8105600993099120273">"Artista"</string>
     <string name="ringtone_default" msgid="1744846699922263043">"Tono de llamada predeterminado"</string>
     <string name="notification_sound_default" msgid="6541609166469990135">"Sonido de notificación predeterminado"</string>
-    <string name="alarm_sound_default" msgid="5488847775252870314">"Sonido de alarma pred."</string>
+    <string name="alarm_sound_default" msgid="5488847775252870314">"Sonido de alarma predeterminado"</string>
     <string name="root_images" msgid="5861633549189045666">"Imágenes"</string>
     <string name="root_videos" msgid="8792703517064649453">"Vídeos"</string>
     <string name="root_audio" msgid="3505830755201326018">"Audio"</string>
diff --git a/res/values-iw/strings.xml b/res/values-iw/strings.xml
index 9d9d918..2de44d3 100644
--- a/res/values-iw/strings.xml
+++ b/res/values-iw/strings.xml
@@ -21,7 +21,7 @@
     <string name="app_label" msgid="9035307001052716210">"אחסון מדיה"</string>
     <string name="artist_label" msgid="8105600993099120273">"אמן"</string>
     <string name="ringtone_default" msgid="1744846699922263043">"רינגטון ברירת מחדל"</string>
-    <string name="notification_sound_default" msgid="6541609166469990135">"צליל ברירת המחדל לתזכורת"</string>
+    <string name="notification_sound_default" msgid="6541609166469990135">"צליל ברירת המחדל להתראות"</string>
     <string name="alarm_sound_default" msgid="5488847775252870314">"צליל לשעון מעורר"</string>
     <string name="root_images" msgid="5861633549189045666">"תמונות"</string>
     <string name="root_videos" msgid="8792703517064649453">"סרטונים"</string>
diff --git a/res/values-pt-rPT/strings.xml b/res/values-pt-rPT/strings.xml
index 9835eda..e53c849 100644
--- a/res/values-pt-rPT/strings.xml
+++ b/res/values-pt-rPT/strings.xml
@@ -21,7 +21,7 @@
     <string name="app_label" msgid="9035307001052716210">"Armazenamento de multimédia"</string>
     <string name="artist_label" msgid="8105600993099120273">"Artista"</string>
     <string name="ringtone_default" msgid="1744846699922263043">"Toque predefinido"</string>
-    <string name="notification_sound_default" msgid="6541609166469990135">"Som notif. predefinido"</string>
+    <string name="notification_sound_default" msgid="6541609166469990135">"Som de notificação predefinido"</string>
     <string name="alarm_sound_default" msgid="5488847775252870314">"Som de alarme predefinido"</string>
     <string name="root_images" msgid="5861633549189045666">"Imagens"</string>
     <string name="root_videos" msgid="8792703517064649453">"Vídeos"</string>
-- 
2.17.1

